ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -469,28 +490,40 @@ public class BottomSheetController: UIViewController {
         bottomSheetOffsetConstraint.constant = -offsetFromBottom
     }
 
-    // MARK: - Height constraint utils
+    private func updateSheetSizingConstraints() {
+        if preferredExpandedContentHeight > 0 {
+            fullScreenSheetConstraint.isActive = false
 
-    private func updateBottomSheetHeightConstraint() {
-        let newConstraint = generateBottomSheetHeightConstraint()
+            // Apply the new preferred height to the static layout guide
+            preferredExpandedContentGuideTopConstraint?.constant = -preferredExpandedContentHeight
 
-        bottomSheetHeightConstraint.isActive = false
-        newConstraint.isActive = true
+            // Activate constraint which ties expandedContentView height to the layout guide height
+            preferredExpandedContentHeightConstraint.isActive = true
 
-        bottomSheetHeightConstraint = newConstraint
+        } else {
+            // Tie the sheet size to maxSheetHeightLayoutGuide.heightAnchor to make it full screen
+            preferredExpandedContentHeightConstraint.isActive = false
```

Comment:
collapsing the bottomsheet case do not need to turn off both of these constraints?

Reply:
No, the sheet is rigid. The height doesn't change as you collapse it.